tensorflow/compiler/mlir/tosa/transforms/legalize_tfl.cc

```diff
@@ -4556,133 +4788,134 @@ void LegalizeTFL::runOnOperation() {
 
 void populateLegalizeTFLPatterns(MLIRContext* ctx,
                                  RewritePatternSet& patterns) {
-#define DEF_PATTERN_INSERT(PAT) \
-  patterns.addWithLabel<Convert##PAT##Op>({#PAT}, ctx);
-
-  DEF_PATTERN_INSERT(TFLAbs);
```

Comment:
Why did you change all of these lines instead of just changing the existing define